test/functional/feature_taproot.py

```diff
@@ -435,8 +436,11 @@ def to_script(elem):
 # - Whether this test demands being placed in a txin with no corresponding txout (for testing SIGHASH_SINGLE behavior)
 
 Spender = namedtuple("Spender", "script,comment,is_standard,sat_function,err_msg,sigops_weight,no_fail,need_vin_vout_mismatch")
+# The full node versions treat it standard V23 means the major version 23.0
```

Comment:
This comment is hard to parse, and it doesn't really explain each value in the enum. Could you please clarify?